src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1823,9 +1884,10 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
         call mpas_pool_get_subpool(block % structs, 'diagnostics', diagnosticsPool)
         call mpas_pool_get_subpool(block % structs, 'scratch', scratchPool)
         ! Compute normalGMBolusVelocity; it will be added to the baroclinic modes in Stage 2 above.
-        if (config_use_standardGM) then
-           call ocn_gm_compute_Bolus_velocity(diagnosticsPool, meshPool, scratchPool)
-        end if
+ !       if (config_use_GM.or.config_use_Redi) then
+ !          call ocn_gm_compute_Bolus_velocity(statePool, diagnosticsPool, &
```

Comment:
@vanroekel I saw that this line is commented out in `ocean/develop`, and looks like we were in a rush and left some confusing code. This was commented out in this PR in:
   66a13d973 fixes for restart test
We changed E3SM to call this routine from the driver:
 ```
components/mpas-ocean/driver/ocn_comp_mct.F:968:
              call ocn_gm_compute_Bolus_velocity(statePool, diagnosticsPool, meshPool, scratchPool, timeLevelIn=1)
```
in https://github.com/E3SM-Project/E3SM/commit/b29cfe5b370 (Luke Van Roekel   2020-05-05 16:01:43 -0500  968)
but it appears that it is not called from the driver in stand-alone. I can make an issue to clean this up, but wanted to ping you on these lines.

Reply:
yes @mark-petersen I removed those calls and moved them to the driver, I just never removed them.  I can clean this up if you want.  It should be in stand alone as you note below